chef/cookbooks/updater/recipes/default.rb

```diff
@@ -40,6 +42,50 @@
       zypper_command += " --auto-agree-with-licenses"
     end
 
+    ruby_block "check for updates" do
+      block do
+        case node[:updater][:zypper][:method]
+        when "patch"
+          command = "list-patches"
+        when "update"
+          command = "list-updates"
+        when "dist-upgrade"
+          command = "list-updates"
+        end
+        system("zypper -q #{command}")
+        # exit 0: no updates, 1: updates
+        node.run_state["needs_update"] = $?.exitstatus ? false : true
+
+        command += '|egrep -q "corosync|pacemaker"'
+        system("zypper #{command}")
+        # exit 0: found, 1 not found
```

Comment:
wait, its not, then something is not working properly in this check, to revise further!